lib/modules/onc_program/bulk_data_group_validation_sequence.rb

```diff
@@ -96,16 +108,16 @@ def check_file_request(file, klass, validate_all, lines_to_validate, must_suppor
         streamed_ndjson_get(file['url'], headers) do |response, resource|
           assert response.headers['Content-Type'] == 'application/fhir+ndjson', "Content type must be 'application/fhir+ndjson' but is '#{response.headers['Content-type']}"
 
-          break if !validate_all && line_count >= lines_to_validate && (klass != 'Patient' || @has_min_patient_count)
+          @patient_ids_seen = Set.new if klass == 'Patient' && @patient_ids_seen.nil?
```

Comment:
I don't really understand this and the other `@patient_ids_seen` initialization.  Can't we just do
```
@patient_ids_seen = Set.new
```
in the `initialize` method?  Do we need to be resetting it along the way?

Reply:
fixed